app/forms/candidate_interface/immigration_right_to_work_form.rb

```diff
@@ -2,43 +2,34 @@ module CandidateInterface
   class ImmigrationRightToWorkForm
     include ActiveModel::Model
 
-    attr_accessor :immigration_right_to_work
+    attr_accessor :right_to_work_or_study, :right_to_work_or_study_details
```

Comment:
The second attribute ('details') could be dropped now, as that's now on a separate form?

```suggestion
    attr_accessor :right_to_work_or_study
```

Reply:
Good spot, have updated ♻️